database/mysql/mysql.go

```diff
@@ -13,15 +13,10 @@ import (
 	nurl "net/url"
 	"strconv"
 	"strings"
-)
 
-import (
 	"github.com/go-sql-driver/mysql"
-	"github.com/hashicorp/go-multierror"
-)
-
-import (
 	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/hashicorp/go-multierror"
```

Comment:
Move this line back up so imports are grouped. e.g. built-in modules -> 3rd party modules -> intra-package imports